src/DssSpell.t.sol

```diff
@@ -1596,4 +1596,28 @@ contract DssSpellTest is DSTest, DSMath {
     function test_auth_in_sources() public {
         checkAuth(true);
     }
+
+    function test_core_unit_budgets() public {
+        address risk = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
+        address multisig = 0x73f09254a81e1F835Ee442d1b3262c1f1d7A13ff;
+
+        uint256 prevSin = vat.sin(address(vow));
+        uint256 prevDai = dai.balanceOf(multisig);
+        uint256 prevRisk = dai.balanceOf(risk);
+
+        assertEq(dai.allowance(address(pauseProxy), address(daiJoin)), 0);
```

Comment:
I don't get this check very well. The one that it would make more sense to check is the `vat.wish` of the `pauseProxy` to the `daiJoin` as it is a `vat.dai` "transferFrom" action the one that happens on `daiJoin.exit`.

Reply:
```
assertEq(vat.can(address(pauseProxy), address(daiJoin)), 1);
```